src/mol-state/transform.ts

```diff
@@ -142,6 +142,16 @@ namespace Transform {
         return { ...t, tags, version: UUID.create22() };
     }
 
+    export function withDependsOn(t: Transform, newDependsOn?: string | string[]): Transform {
```

Comment:
What is the purpose of this function?

Reply:
Updates depends on field of the transform object